routes/account/index.js

```diff
@@ -12,63 +12,63 @@ const UserCollections = require('models/User');
 const Utils = require('common/Utils')()
 
 router.get('/login/success', function(req, res) {
-	res.redirect(303, '/');
+  res.redirect(303, '/');
 });
 
 router.post(
-	'/login',
-	localPassport.authenticate(
-		'local', {
-			successRedirect: '/',
-			failureRedirect: '/account/fail'
-		}
-	)
+  '/login',
+  localPassport.authenticate(
+    'local', {
+      successRedirect: '/',
+      failureRedirect: '/account/fail'
+    }
+  )
 );
 
 router.post('/logout', function(req, res) {
-	delete req.session.passport.user;
-	res.send('ログアウト完了しました。');
+  delete req.session.passport.user;
+  res.send('ログアウト完了しました。');
 });
 
 router.get('/verify', function(req, res) {
-	let verifyId = req.query.id
-	let conditions = { verifyId: verifyId }
-	let updates = { isVerified: true }
-	let options = {
-		new: true,
-		upsert: true
-	}
-	UserCollections.findOneAndUpdate(conditions, updates, options, function(err, savedUser){
-		if(err){
-			Utils.postSlack(err)
-			return res.status(403).send('認証は拒否されました。')
-		}
-		Utils.postSlack('[' + savedUser.mail + '] が新規ユーザーとして認証されました。')
-		res.redirect(303, '/account/login')
-	})
+  let verifyId = req.query.id
+  let conditions = { verifyId: verifyId }
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

*  Missing semicolon.